src/components/firstProfile/Status.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from 'react'
+import Logo from '../logo/Logo'
+import { Typography, Box } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import PrevPageButton from '../../common/buttons/PrevPageButton'
+import NextPageButton from '../../common/buttons/NextPageButton'
+import { commonStyles } from 'styles/commonStyles'
+
+const STATUSES = [
+  'Looking for new friends',
+  "I’m learning a new language. Let's talk!",
+  "Let's be friends, I'm new in town",
+  'I look for emotional support',
+  'My plans are to move abroad',
+  "I'm a new mom. Need some help",
+  "Let's take the dogs for a walk",
+]
+
+const Status = () => {
+  const { classes } = useStyles()
+  const commonClasses = commonStyles().classes
+  const [userStatus, setUserStatus] = useState<number[]>([])
+
+  const statusHandler = (event: React.MouseEvent) => {
```

Comment:
`const [selectedStatuses, setSelectedStatuses] = useState([]);

  const toggleStatus = (status) => {
    if (selectedStatuses.includes(status)) {
      setSelectedStatuses(selectedStatuses.filter((s) => s !== status));
    } else {
      if (selectedStatuses.length < 3) {
        setSelectedStatuses([...selectedStatuses, status]);
      } else {
        // Alert or some indication that user can only select up to 3 statuses
        alert("You can only select up to 3 statuses.");
      }
    }
  };`
  
i would suggest to use GPT option - it's a bit easier. In this case you don't need  `checkStatus`.
Instead you will have further condition:
`backgroundColor: selectedStatuses.includes(status)
               ? '#faa06d'
                : 'rgba(229, 229, 229, 0.40)'`

Reply:
to implement this I would need to create another state status which would be used in the suggested backgroundColor line. If that's okay, I will rewrite this part of the code